### DISCUSSION TITLE
What is the easiest (or recommended) way to integrate an existing Svelte app into a Wails app?

### DISCUSSION THREAD

**Discussion by serge-hulne** [wailsapp/wails#2298] [Q&A] [answered]
1. Manually copy the .svelte files into  the Wails project (in `frontend/src/`).
2. Just overwrite the `frontend/src` dir with the content of the `src` dir of the Svelte app?
3. Some other way?

**Comment by leaanthony** [comment, marked as answer]
The answer is "any way you want to"... The only things you need to do are:
- ensure you embed the compiled project assets
- update `wails.json` with your project directory and build commands

**Reply by serge-hulne** [reply]
Thank you!
Congratulations for creating Wails, I find it useful, elegant, and efficient.
